string topics
string dest
---
bool value
